execution/aggregate/accumulator.go

```diff
@@ -287,6 +309,7 @@ func (c *countAcc) Reset(_ float64) {
 }
 
 type avgAcc struct {
+	ctx         context.Context
```

Comment:
can we just add it to the "Add" and "AddVector" method as first argument instead as a struct field of the accumulator?